src/transformState.js

```diff
@@ -46,7 +46,27 @@
  * @param {Object[]} transforms
  */
 function transformState(state, transforms) {
-  // write code here
+  for (const action of transforms) {
```

Comment:
that's not an action that's an object with action and properties
```suggestion
  for (const transform of transforms) {
```